ccloud-monitoring/docs/producer-use-cases/connectivity-problem.rst

```diff
@@ -0,0 +1,86 @@
+.. _ccloud-cli-tutorial-producer-connectivity-problem:
+
+Connectivity Problem
+********************
+In this use case we will simulate a network failure, see how your producer reacts when it can't reach the broker.
```

Comment:
Just throwing this idea out there to see if it sticks...but regarding the intent to highlight relevant MBeans, would it make sense to mention which MBeans (JMX metric) each scenario will hit, at the top of each scenario?

Reply:
Gotcha, sounds like a good idea.

Reply:
Used jconsole to find the mbean object for the metrics used in the producer case (only place mbean attributes are used) and updated the docs accordingly.